Scientific_workflows/FractionalCoverOfWaterProject/Manual_classification_workflow.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
<p>Please comment this code - hard to tell what it does as is. Might also be nice to explain what you're doing as you go</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/663/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done</p>